src/internal/ppsdb/ppsdb.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package pps_v2;
+option go_package = "github.com/pachyderm/pachyderm/v2/src/internal/ppsdb";
+
+import "pps/pps.proto";
+
+message ClusterDefaultsWrapper {
+  reserved 1, 2;
+  reserved "details_json", "effective_details_json";
```

Comment:
why reserve this?

Reply:
Because in v2.7, this was a message with field 1 named `details_json` & field 2 named `effective_details_json`; it was written to the database but never actually used. Reserving the fields lets marshalling & unmarshalling work without an error (and they can be ignored because they weren’t actually used). It’s purely a compatibility measure.